atsign-foundation/at_widgets
# at_app_flutter - Remove AtContext model

**- What I did**

- Removed the AtContext model, and replaced it with the rsdk AtClientManager

**- How I did it**

**- How to verify it**

See the [at_app verification process](https://github.com/atsign-foundation/at_app/pull/20)

**- Description for the changelog**
at_app_flutter - Remove AtContext model
